### PR TITLE
#1 Feature: Frictionless -> ckan

### PR DESCRIPTION
**Pull request by kmanaseryan**
# Description

- Implemented algorithm for converting **frictionless->ckan**

Refs #1

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules E

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
 
+![Node.js CI](https://github.com/datopian/frictionless-ckan-mapper-js/workflows/Node.js%20CI/badge.svg)
+
 This is JS version of [Frictionless CKAN Mapper](https://github.com/frictionlessdata/frictionless-ckan-mapper).
 
 A library for mapping CKAN metadata <=> Frictionless metadata.
+
+# Sources
+
+- [Frictionless - Mappings from Other Formats to Frictionless both Profiles and Schemas](https://docs.google.com/spreadsheets/d/1XdqGTFni5Jfs8AMbcbfsP7m11h9mOHS0eDtUZtqGVSg/edit#gid=1925460244)
```
Comment (mariorodeghiero) [inline README.md:11]: @kmanaseryan The code looks good :clap:  but the documentation of the README I think is small. **Note** I don't know if you will improve this in the future, will be?

i.e I'm not up to date with this, so I would like to do:

- How to install
- How to run/use
- How to run tests

**Comment by mariorodeghiero** [comment]
@kmanaseryan All the tests work properly locally for me :heavy_check_mark: :clap:

File: README.md (hunk 1 above)
Comment (kmanaseryan) [reply README.md:11]: Yea absolutely agree! I will improve the docs and will add all your points too👍

Comment (mariorodeghiero) [reply README.md:11]: @kmanaseryan sounds good.
I have an example here https://github.com/datopian/ckan-client-js#use if you want to follow someone.

Comment (kmanaseryan) [reply README.md:11]: @mariorodeghiero I've seen this and actually I added in my list as a nice example of documentation and not only. Very clean and descriptive, thank you!👍

**Pull request merged**